jib-core/src/main/java/com/google/cloud/tools/jib/json/JsonTemplateMapper.java

```diff
@@ -63,6 +67,29 @@
     return objectMapper.readValue(Files.newInputStream(jsonFile), templateClass);
   }
 
+  /**
+   * Deserializes a JSON file via a JSON object template.
+   *
+   * @param <T> child type of {@link JsonTemplate}
+   * @param jsonFile a file containing a JSON string
+   * @param templateClass the template to deserialize the string to
+   * @param acquireLock if true then acquire a shared lock on the file
+   * @return the template filled with the values parsed from {@code jsonFile}
+   * @throws IOException if an error occurred during reading the file or parsing the JSON
+   */
+  public static <T extends JsonTemplate> T readJsonFromFile(
+      Path jsonFile, Class<T> templateClass, boolean acquireLock) throws IOException {
+    if (!acquireLock) {
+      return objectMapper.readValue(Files.newInputStream(jsonFile), templateClass);
```

Comment:
I think here we can either:
- call out to the method without the extra `acquireLock` parameter, or
- remove that other method without the `acquireLock` parameter, or
- remove the `acquireLock` parameter and rename this method into something like `readJsonFromFileWithLocking`